### PR TITLE
update(besu,quorum): adapt besu & quorum ansible tasks and template files to charts

### PR DESCRIPTION
**Pull request by alvaropicazo**
This PR contains:

 - Changes in ansible tasks from quorum & besu
 - Changes in tpl files from quorum & ansible
 - Fix in besu configmap for transaction pool max capacity size variable

### CODE REVIEW

File: platforms/hyperledger-besu/charts/besu-node/templates/besu-config-configmap.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ data:
     node-private-key-file={{.Values.node.besu.privateKeyPath | quote }}
 
     # Transaction Pool
-    tx-pool-layer-max-capacity={{ .Values.node.besu.txPool.maxCapacity }}
+    tx-pool-max-size={{ .Values.node.besu.txPool.maxCapacity }}
```
Comment (suvajit-sarkar) [inline platforms/hyperledger-besu/charts/besu-node/templates/besu-config-configmap.yaml:30]: why are we changing this ?

File: platforms/quorum/charts/quorum-genesis/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# apiVersion: v2
```
Comment (suvajit-sarkar) [inline platforms/quorum/charts/quorum-genesis/Chart.yaml:1]: why change the apiVersion ?

**Review by suvajit-sarkar — COMMENTED** [review]
Rest LGTM

File: platforms/hyperledger-besu/charts/besu-node/templates/besu-config-configmap.yaml (hunk 1 above)
Comment (alvaropicazo) [reply platforms/hyperledger-besu/charts/besu-node/templates/besu-config-configmap.yaml:30]: Due to the changes of the quorum version. They have renamed that variable

Comment (alvaropicazo) [reply platforms/hyperledger-besu/charts/besu-node/templates/besu-config-configmap.yaml:30]: Due to the changes of the quorum version. They have renamed that variable

File: platforms/quorum/charts/quorum-genesis/Chart.lock [hunk 3]
```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: bevel-vault-mgmt
```
Comment (sownak) [inline platforms/quorum/charts/quorum-genesis/Chart.lock:2]: don't checkin lock files

**Review by sownak — CHANGES_REQUESTED** [review]
Few changes needed

**Pull request merged**